test/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests/ConfigurationManagerTests.cs

```diff
@@ -750,6 +718,96 @@ public void TestConfigurationComparer()
             TestUtilities.AssertFailIfErrors(context);
         }
 
+        [Fact]
+        public async Task RequestRefresh_RespectsRefreshInterval()
```

Comment:
do we need to check for extremely short or long durations on refresh interval?

Reply:
Added a few more scenarios, used some better names.